### PR TITLE
Bitbucket: clarify that OAuth1 can not longer be user since Server/DataCenter v8

### PR DESCRIPTION
**Pull request by glaubinix**

### CODE REVIEW

File: docs/cloud/bitbucket-server-oauth1-integration-setup.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Bitbucket Data Center / Server Integration Setup (OAuth 1)
 ## Private Packagist Cloud
 
+<div class="row column">
+    <div class="callout warning">
+        <p>
+            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.
+            Consult the <a href="/docs/cloud/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.
+            If you have an existing OAuth 1 Application Link, create an OAuth 2 Application Link and edit the existing integration in Private Packagist.
```
Comment (naderman) [inline docs/cloud/bitbucket-server-oauth1-integration-setup.md:9]:
```suggestion
            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```

File: docs/self-hosted/bitbucket-server-oauth1-integration-setup.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 # Bitbucket Data Center / Server Integration Setup (OAuth 1)
 ## Private Packagist Self-Hosted
 
+<div class="row column">
+    <div class="callout warning">
+        <p>
+            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.
+            Consult the <a href="/docs/self-hosted/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.
+            If you have an existing OAuth 1 Application Link, create an OAuth 2 Application Link and edit the existing integration in Private Packagist.
```
Comment (naderman) [inline docs/self-hosted/bitbucket-server-oauth1-integration-setup.md:9]:
```suggestion
            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```

**Review by naderman — CHANGES_REQUESTED** [review]

File: docs/cloud/bitbucket-server-oauth1-integration-setup.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
 # Bitbucket Data Center / Server Integration Setup (OAuth 1)
 ## Private Packagist Cloud
 
+<div class="row column">
+    <div class="callout warning">
+        <p>
+            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.
+            Consult the <a href="/docs/cloud/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.
+            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```
Comment (pscheit) [inline docs/cloud/bitbucket-server-oauth1-integration-setup.md:9]:
```suggestion
            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.<br>
            Consult the <a href="/docs/cloud/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.<br>
            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```
I guess you wanted that?

File: docs/self-hosted/bitbucket-server-oauth1-integration-setup.md [hunk 4]
```diff
@@ -1,6 +1,16 @@
 # Bitbucket Data Center / Server Integration Setup (OAuth 1)
 ## Private Packagist Self-Hosted
 
+<div class="row column">
+    <div class="callout warning">
+        <p>
+            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.
+            Consult the <a href="/docs/self-hosted/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.
+            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```
Comment (pscheit) [inline docs/self-hosted/bitbucket-server-oauth1-integration-setup.md:9]:
```suggestion
            Note: As of Bitbucket Data Center / Server v8 Application Links using OAuth 1 can no longer be used to set up synchronizations in Private Packagist.<br>
            Consult the <a href="/docs/self-hosted/bitbucket-server-integration-setup.md">Bitbucket Data Center / Server OAuth 2 setup guide</a> to set up an Application Link using OAuth 2 instead.<br>
            If you have an existing OAuth 1 Application Link, do not delete the Private Packagist integration. Instead create an OAuth 2 Application Link on the Bitbucket side but edit the existing integration in Private Packagist to use the new OAuth 2 Application Link.
```

**Review by pscheit — APPROVED** [review]

File: docs/cloud/bitbucket-server-oauth1-integration-setup.md (hunk 3 above)
Comment (glaubinix) [reply docs/cloud/bitbucket-server-oauth1-integration-setup.md:9]: Not really. I just added new lines to make it easier work with. If you think that the line breaks help with reading it in a web page then let's add them

Comment (glaubinix) [reply docs/cloud/bitbucket-server-oauth1-integration-setup.md]: Added the second line break but removed the first

**Pull request merged**